public/content/developers/docs/consensus-mechanisms/pow/index.md

```diff
@@ -7,7 +7,7 @@ lang: en
 The Ethereum network began by using a consensus mechanism that involved **[Proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow)**. This allowed the nodes of the Ethereum network to agree on the state of all information recorded on the Ethereum blockchain and prevented certain kinds of economic attacks. However, Ethereum switched off proof-of-work in 2022 and started using [proof-of-stake](/developers/docs/consensus-mechanisms/pos) instead.
 
 <InfoBanner emoji=":wave:">
-    Proof-of-work has now been deprecated. Ethereum no longer uses proof-of-work as part of its consensus mechanism. Instead, it uses proof-of-stake. Read more on <a href="/developers/docs/consensus-mechanisms/pos/">proof-of-stake</a> and <a href="/staking/">staking</a>.
+    Proof-of-work has now been deprecated. Ethereum no longer uses proof-of-work as part of its consensus mechanism. Instead, it uses proof-of-stake. Read more on [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) and [staking](/staking/).
```

Comment:
Did this change with our Next.js mdx updates? We previously were forced to use html when inside JSX components since the MD wouldn't parse, but looks like this is working great 👍  Much cleaner

Reply:
Im not entirely sure why this changed, but was working for me so just went for it